### PR TITLE
Release foreman_salt 2.1 (RPM)

### PR DESCRIPTION
**Pull request by stbenjam**

### CODE REVIEW

**Comment by stbenjam** [comment]
For 1.8 and nightly

File: rubygem-foreman_salt/rubygem-foreman_salt.spec [hunk 1]
```diff
@@ -14,10 +14,11 @@
 %define rubyabi 1.9.1
 %global foreman_dir /usr/share/foreman
 %global foreman_bundlerd_dir %{foreman_dir}/bundler.d
+%global foreman_pluginconf_dir %{foreman_dir}/config/settings.plugins.d
```
Comment (domcleal) [inline rubygem-foreman_salt/rubygem-foreman_salt.spec]: unused?  You should be able to remove all three definitions now.

**Comment by stbenjam** [comment]
@domcleal updated

**Comment by domcleal** [comment]
Thanks @stbenjam!

**Pull request merged**